tests/console/yast2_lan.pm

```diff
@@ -16,6 +16,7 @@ use base "console_yasttest";
 use strict;
 use testapi;
 use utils;
+use version_utils qw(is_sle is_tumbleweed is_leap);
```

Comment:
Are you using is_tumbleweed anywhere? Could not see it (from my mobile at least ;) )

Reply:
Good catch, I didn't need it in the end, but didn't remove it from the imports. Will remove with the next PR.